modules/recording/recording/recording.tsx

```diff
@@ -125,4 +114,10 @@ const Recording = () => {
   );
 };
 
-GlobalCore.manager.app("recording", Recording);
+GlobalCore.manager.app("recording", () => {
+  return (
+    <Provider store={store}>
```

Comment:
Im not sure if this can go to the `core` part, wrapping whatever component we appen to it?